### PR TITLE
JW-63 Duplicate values in system PATH

### PR DESCRIPTION
**Pull request by mmasztalerczuk**
-> Added removing duplicated values
  -> Removed problem with not existing 'vendor' branch
  -> Some small changes during git instalation

### CODE REVIEW

File: ci/install_git.ps1 [hunk 1]
```diff
@@ -2,14 +2,28 @@
 
 # according to:
 # https://docs.docker.com/opensource/project/software-req-win/
+Write-Host ""
```
Comment (m-kostrzewa) [inline ci/install_git.ps1]: I think we should stick to echo or Write-Host.

**Review by m-kostrzewa — CHANGES_REQUESTED** [review]
use echo, otherwise approve

**Comment by mmasztalerczuk** [comment]
I replace _Write-Host_ in one place, but I need _Write-Host_ because _echo_ can't set color and write to input without new line char.

**Comment by m-kostrzewa** [comment]
ok

**Review by m-kostrzewa — APPROVED** [review]

**Review by Attil — APPROVED** [review]

**Comment by unya** [comment]
LGTM. Merging

**Pull request merged**